### ISSUE TITLE
Feel++ 0.100.0 Beta 1 release doesn't compile on OS X

### ISSUE THREAD

**Issue by aancel** [feelpp/feelpp#519]
Following issue https://github.com/feelpp/feelpp/issues/513, I tried to update the homebrew package for feel++, but it does not compile.
My guess is that it comes from this commit https://github.com/feelpp/feelpp/commit/4dd247cd2c488a6d1a52e2a19e2ba7656e372d5a, that has not been merged in the release. I managed to reproduced the error, while building manually from the tarball of the release.

**Comment by aancel** [comment]
@prudhomm 
- I wanted to check if this issue was solved with the newer beta 5 package of Feel++, but there seem to be an new issue with this version and nlopt. Nlopt causes the cmake process to fail because it cannot clone it from git (there are no .git directory in the package).
I also noticed that nlopt was packaged in beta 1 in `contrib/nlopt`, but not in beta 5. It seems to be the cause of the failure.

- Secondly, I tried to disable nlopt with `-DFEELPP_ENABLE_NLOPT=OFF`, but an other problem happens with quickstart (which in my opinion is similar to the one with nlopt). At the configuration step, cmake will try to do a clone of the newly created quickstart repository, but will fail with a similar error to the one hapenning with nlopt. I guess quickstart also has to be packaged in the archive.

- Lastly, there is a final `git.*.log`written the root source dir, regarding hpddm, with the same error in it. This is probably the same problem as the two previous points.
